# Examining Racial Discrimination in the US Job Market

### Background
Racial discrimination continues to be pervasive in cultures throughout the world. Researchers examined the level of racial discrimination in the United States labor market by randomly assigning identical résumés to black-sounding or white-sounding names and observing the impact on requests for interviews from employers.

### Data
In the dataset provided, each row represents a resume. The 'race' column has two values, 'b' and 'w', indicating black-sounding and white-sounding. The column 'call' has two values, 1 and 0, indicating whether the resume received a call from employers or not.

Note that the 'b' and 'w' values in race are assigned randomly to the resumes when presented to the employer.

### Exercises
You will perform a statistical analysis to establish whether race has a significant impact on the rate of callbacks for resumes.

Answer the following questions **in this notebook below and submit to your Github account**. 

   1. What test is appropriate for this problem? Does CLT apply?
   2. What are the null and alternate hypotheses?
   3. Compute margin of error, confidence interval, and p-value.
   4. Write a story describing the statistical significance in the context or the original problem.
   5. Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?

You can include written notes in notebook cells using Markdown: 
   - In the control panel at the top, choose Cell > Cell Type > Markdown
   - Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet


#### Resources
+ Experiment information and data source: http://www.povertyactionlab.org/evaluation/discrimination-job-market-united-states
+ Scipy statistical methods: http://docs.scipy.org/doc/scipy/reference/stats.html 
+ Markdown syntax: http://nestacms.com/docs/creating-content/markdown-cheat-sheet
****

In [4]:
import pandas as pd
import numpy as np
from scipy import stats

In [5]:
data = pd.io.stata.read_stata('us_job_market_discrimination.dta')

In [7]:
# number of callbacks for black-sounding names
sum(data[data.race=='b'].call)


157.0

In [9]:
# number of callbacks for white-sounding names
sum(data[data.race == 'w'].call)

235.0

In [12]:
data.groupby('call').size()

call
0.0    4478
1.0     392
dtype: int64

In [13]:
data.head()

id ad  education  ofjobs  yearsexp  honors  volunteer  military  empholes  \
0  b  1          4       2         6       0          0         0         1   
1  b  1          3       3         6       0          1         1         0   
2  b  1          4       1         6       0          0         0         0   
3  b  1          3       4         6       0          1         0         1   
4  b  1          3       3        22       0          0         0         0   

   occupspecific    ...      compreq  orgreq  manuf  transcom  bankreal trade  \
0             17    ...          1.0     0.0    1.0       0.0       0.0   0.0   
1            316    ...          1.0     0.0    1.0       0.0       0.0   0.0   
2             19    ...          1.0     0.0    1.0       0.0       0.0   0.0   
3            313    ...          1.0     0.0    1.0       0.0       0.0   0.0   
4            313    ...          1.0     1.0    0.0       0.0       0.0   0.0   

  busservice othservice  missind  ownership  
0        0.0        0.0      0.0             
1        0.0        0.0      0.0             
2        0.0        0.0      0.0             
3        0.0        0.0      0.0             
4        0.0        1.0      0.0  Nonprofit  

[5 rows x 65 columns]

In [15]:
len(data)

4870

1.What test is appropriate for this problem? Does CLT apply?

ANSWER: We are comparing two independent groups and therefore a 2 - sample t-test. Since the two groups are distinct and if we use a paired t-test which would be the same the one sample t-test then we use the results of the pre-test for the post-test group. So heence the groups are distinct here two sample t-test applies. 

Also the CLT will apply since there are 4870 observations

2.What are the null and alternate hypotheses?

ANSWER: We have the following
h0: Callback for the black sounding names - Callback for the white sounding name ==0
h1: Callback for the black sounding names - Callback for the white sounding name !=0




3.Compute margin of error, confidence interval, and p-value.

ANSWER

In [17]:
black_calls = data.call[data.race == 'b']
white_calls= data.call[data.race == 'w']
stats.ttest_ind(black_calls,white_calls)

Ttest_indResult(statistic=-4.1147052908617514, pvalue=3.9408021031288859e-05)

In [21]:
xbar_b = black_calls.mean()
sd_b = black_calls.std()    

xbar_w = white_calls.mean()
sd_w = white_calls.std()

sample_statistic = xbar_b - xbar_w
se = (((sd_b)**2/len(black_calls)) + ((sd_w)**2/len(white_calls)))**(1/2)
me = 1.96*se
plus = sample_statistic + me
minus = sample_statistic - me 
print (' 95% Confidence Interval: {minus} to {plus} \n Margin of Error: {me}'.format(minus=minus,plus=plus, me=me))

 95% Confidence Interval: -0.04729127241789561 to -0.016774437292225546 
 Margin of Error: 0.015258417562835034


4.Write a story describing the statistical significance in the context or the original problem.


ANSWER:
Assumption is that there is racial discrimination and in the United states there was an experiment that was done that assigned black sounding names or white sounding names to resumes. In the study they measured if the resumes got a callback from employers and if the white sounding names or black sounding names was significantly different.


5.Does your analysis mean that race/name is the most important factor in callback success? Why or why not? If not, how would you amend your analysis?

ANSWER: No the racial discrimination is not proved by this test as we need to consider other multiple variables instead of just the callbacks.